src/main/java/it/gov/pagopa/pu/sil/service/AuthorizationService.java

```diff
@@ -26,6 +28,11 @@ public AuthorizationService(AuthnClient authClientImpl) {
     this.authClientImpl = authClientImpl;
   }
 
+  public AccessToken requestLimitedToken(LimitedTokenRequest limitedTokenRequest, String accessToken) {
+    log.info("Requesting limited token for {}", limitedTokenRequest);
```

Comment:
in case of many DP, this could lead to too many logs not so relevant
```suggestion
    log.debug("Requesting limited token for {}", limitedTokenRequest);
```